pkg/api/types_test.go

```diff
@@ -5269,3 +4859,99 @@ func TestDcosConfigHasBootstrap(t *testing.T) {
 		}
 	}
 }
+
+func TestKubernetesAddonIsEnabled(t *testing.T) {
```

Comment:
This unit test reflects the new, simpler "IsEnabled" method for an addon. There is no longer any "if nil override" parameter context.